core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/EntityStatisticsCacheMonitor.scala

```diff
@@ -38,16 +40,28 @@ class EntityStatisticsCacheMonitorActor(val dataSource: SlickDataSource, val tim
 
   override def preStart(): Unit = {
     super.preStart()
-    self ! Sweep
+    self ! Start
   }
 
   override def receive = {
+    case Start => sender ! Sweep
```

Comment:
Why do we need this additional Start state before the Sweep state?

Reply:
heh, good catch/good question. We don't really need it. I added it to make unit testing easier - in the unit tests, I really wanted a way to test if an Actor was responding, and the easiest way to do that was to have some case where if I sent the Actor a message, it would respond with a message. So I added the `Start` -> `Sweep` here, and use it at https://github.com/broadinstitute/rawls/pull/1569/files#diff-d88e8e2a2ea10f6bc99fc697520db131fa10a43e0591bde5357749caa0b1a2f7R216-R217.